x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/index.tsx

```diff
@@ -221,7 +225,7 @@ export const AllRules = React.memo<{
             <EuiBasicTable
               columns={columns}
               isSelectable={!hasNoPermissions ?? false}
-              itemId="rule_id"
+              itemId="id"
```

Comment:
nit: It seems a bit generic here to me, would it be clearer to name it something specific like detection_engine_rule_id?

Reply:
it's a value taken from the API